doc/Analytics.xml

```diff
@@ -1788,7 +1788,7 @@
                     <para>fc: AccessoryDescription</para>
                   </entry>
                   <entry>
-                    <para>N/A</para>
+                    <para>fc:HatType</para>
```

Comment:
Suggestion : "fc:subtype" includes "fc:HatType" string

just to make sure the documentation provides reference to new "subtype" element.

Reply:
Again I agree with @bsriramprasad suggestion.